docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,147 @@
+
+We have named this application `todo-backend`. This may be changed to a different name, if desired.
+
+The application will now begin to build, and can be observed via:
+```
+$ oc get build -w
+```
+Once the build is complete, deployment will begin:
+```
+$ oc get deployment todo-backend -w
+```
+Once deployment is complete, we can test the application:
+```
+$ oc get route todo-backend -o jsonpath="{.spec.host}"
+```
+Expected output:
+```
+todo-backend-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+You should now be able to access your application using, for example
+```
+https://todo-backend-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+You can use the todo-frontend to test out the deployed app using:
+```
+https://todobackend.com/client/index.html
+```
+and entering your application URL from above.
+
+If you wish to remove the application when you are done with it, simply run:
+```
+$ helm delete todo-backend
+```
+
+## Example: jaxrs-client
+The second example to build and deploy to OpenShift is the
+https://github.com/wildfly/quickstart/tree/main/jaxrs-client[jaxrs-client] quickstart.
+
+First of all, clone the quickstart repo:
+```
+$ git clone https://github.com/wildfly/quickstart.git
+cd quickstart/jaxrs-client
+```
+
+WildFly Helm Charts is used to build and deploy `jaxrs-client`. The following command install a Helm Chart in a OpenShift cluster (users can connect to their sandbox following the instructions already provided above):
+```
+$ helm install jaxrs-client-from-chart -f charts/helm.yaml wildfly/wildfly
+```
+
+This time, the name of the Helm Chart app is `jaxrs-client-app` (of course, this may be changed to a different name, if desired)
+
+The application will now begin to build, and can be observed via:
+```
+$ oc get build -w
+```
+Once the build is complete, to follow the deployment of the application, run:
+```
+$ oc get deployment jaxrs-client-app -w
+```
+Once deployment is complete, we need to define a route to expose the application on the internet. To do so, run:
+```
+$ oc get route jaxrs-client-app -o jsonpath="{.spec.host}"
+```
+The above command returns back a public address that points to the application deployed to the OpenShift cluster of the user (for example):
+```
+jaxrs-client-app-someuser-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+```
+At this point, defining a env variable for simplicity, it is possible to test the REST endpoint that has been deployed into the OpenShift cluster:
+```
+$ export ROUTE=$(oc get route jaxrs-client-app -o jsonpath="{.spec.host}")
+```
+and use curl to send HTTP method requests to the REST endpoint, for example (GET)
+```
+$ curl -L $ROUTE/rest/contacts
+```
+If you wish to remove the application when you are done with it, simply run:
```

Comment:
Maybe we could add a couple sentences here about the wildfly-maven-plugin configuration? Maybe something like:

Take a look at the wildfly-maven-plugin configuration in the jaxrs-client quickstart's [pom.xml](https://github.com/wildfly/quickstart/blob/main/jaxrs-client/pom.xml#L274-L299). Notice that it specifies the layer that should be used when provisioning WildFly. For more details about this, take a look at the `wildfly-maven-plugin` [documentation](https://docs.wildfly.org/wildfly-maven-plugin/).

Reply:
This is what is missing also on QS READMEs, maybe ping @jfdenise , I guess he may have already something somewhere?